### DISCUSSION TITLE
What tools do you use for annotation?

### DISCUSSION THREAD

**Discussion by ZEROdcFRAME** [au1206/paper_annotations#4] [Q&A] [answered]
Hi au1206, I am ML engineer of recommendation and NLP, this is a nice project!! 

I'd like to contribute by adding annotated, easy-to-read papers, what tools do you use for annotation?

I'd love to send you a pull request.

**Comment by au1206** [comment, marked as answer]
Hi, i am glad you liked the project. I use a software called notability to create the annotation. Would appreciate the contributions but please ensure the style of annotation is identical to maintain uniformity. Looking forward to it.
